Project.toml

```diff
@@ -12,7 +12,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 [compat]
 ColorVectorSpace = "0.8"
 Colors = "0.11, 0.12"
-FreeType = "3"
+FreeType = "3, 4"
```

Comment:
i think this is only compatible with 4 now, right?

Reply:
Oops!